File: src/com/funivan/idea/editUsages/Structures/ReplacesItemsComparator.java
package com.funivan.idea.editUsages.Structures;

import java.util.Comparator;

/**
 * @author Ivan Shcherbak <dev@funivan.com>
 */
public class ReplacesItemsComparator implements Comparator<ReplaceStructure> {

    @Override
    public int compare(ReplaceStructure o1, ReplaceStructure o2) {
        return (o1.getLine() < o2.getLine()) ? 1 : -1;
    }
}

File: src/com/funivan/idea/editUsages/structures/ReplacesItemsComparator.kt
package com.funivan.idea.editUsages.structures

import java.util.*

/**
 * @author Ivan Shcherbak dev@funivan.com
 */
class ReplacesItemsComparator : Comparator<ReplaceStructure> {
    override fun compare(o1: ReplaceStructure, o2: ReplaceStructure): Int {
        return if (o1.line < o2.line) 1 else -1
    }
}
